source/_posts/2017-06-14-Ansible-via-python.markdown

```diff
@@ -0,0 +1,188 @@
+# Ansible via Python ![alt text](https://cub.nobleprog.com/sites/hitramx/files/styles/height50_scale/public/category_image/cursos-de-ansible-en-mexico.png?itok=xPUrGNrA)
+
+**_Ansible is designed around the way people work and the way people work together_**
+
+## What is Ansible
+
+Ansible is simple IT engine for automation, it is designed for manage many
+systems, rather than just one at a time. Ansible automates cloud provisioning,
+configuration management, application deployment, intra-service
+orchestration, and many other IT operations. It is easy to deploy due to using
+no agents and no additional custom security infrastructure. We can define own
+configuration management in simple YAML language, which is named
+ansible-playbook. YAML is easier to read and write by humans than other common
+data formats line XML or JSON. Futhermore, most programming languages contain
+libraries which operate and work YAML.
+
+## Inventory
+
+Ansible works collaterally on many systems in your infrastructure, so it is
```

Comment:
@macpijan This means, that Ansible services systems in paralell way, using only one set of tasks.